main.py

```diff
@@ -1,15 +1,38 @@
+from datetime import datetime, timedelta
+
 import click
 
+from crawl import full_crawl
+
 
 @click.group()
 def cli():
   """Entry point for CLI"""
   pass
 
 @click.command()
-def crawl():
-  """stub of crawl command"""
-  click.echo('crawl command called')
+@click.option('--start', help = 'specify past timestamp to crawl posts to')
+def crawl(start):
+  "Fetch posts within specified timeframe"
```

Comment:
[Docstring nit] Triple quote fix again

```suggestion
  """Fetch posts within specified timeframe"""
```